ocs_ci/ocs/resources/pod.py

```diff
@@ -143,6 +145,58 @@ def exec_ceph_cmd(self, ceph_cmd, format='json-pretty'):
             return [item for item in out if item]
         return out
 
+    def run_io(
+        self, storage_type, size, direction='rw', workers=1, runtime=60,
+        run_in_bg=True
+    ):
+        """
+        Execute FIO on a pod
+
+        Args:
+             storage_type (str): 'fs' or 'block'
+             size (str): Size in MB, e.g. '200M'
+             direction (str): Determines the operation:
+                'ro', 'wo', 'rw' (default: 'rw')
+            workers (int): Number of threads to execute FIO
```

Comment:
fio has numjobs that invokes multiple jobs, we should use that instead of our python threads.

Reply:
Good catch!